core/src/io/anuke/mindustry/world/blocks/power/PowerNode.java

```diff
@@ -221,7 +226,20 @@ public void drawPlace(int x, int y, int rotation, boolean valid){
         Draw.color(Pal.placing);
         Drawf.circles(x * tilesize + offset(), y * tilesize + offset(), laserRange * tilesize);
 
-        getPotentialLinks(tile, other -> Drawf.square(other.drawx(), other.drawy(), other.block().size * tilesize / 2f + 2f, Pal.place));
+        getPotentialLinks(tile, other -> {
+            Drawf.square(other.drawx(), other.drawy(), other.block().size * tilesize / 2f + 2f, Pal.place);
+
+            insulators(tile, other).each(cause -> {
+
+                Block block = world.tileWorld(cause.drawx(), cause.drawy()).block();
```

Comment:
This can be null (so don't call `block()` without a check first), and use `ltileWorld` instead of checking whether it's a blockpart manually.